src/store/page/page-element.ts

```diff
@@ -157,69 +183,68 @@ export class PageElement {
 	 * Adds a child element to is element (and removes it from any other parent).
 	 * @param child The child element to add.
 	 * @param index The 0-based new position within the children.
+	 * @param slotId The slot to attach the element to. When undefined the default slot is used.
```

Comment:
`This indicates the slot, that the element will be attached to. When undefined, the default slot will be used instead.`